docs/CUSTOM_BOARDS_TARGETS.md

```diff
@@ -0,0 +1,61 @@
+# mLRS Documentation: Custom Boards & Targets #
+
+([back to main page](../README.md))
+
+While mLRS offers a variety of hardware options, you may find it beneficial for your application to create a custom board to better suit your objectives.  This page describes how to use the existing codebase to enable the use of custom boards and custom targets.
+
+## Definitions
+
+- 'Board' refers to the physical hardware
+- 'Target' refers to the firmware that will run on the board
+
+## Assumptions
+
+1. You have setup the build environment for mLRS
+2. You are comfortable with C/C++ programming
+
+## Creating a target for a custom board
+
+There are a number of ways of how to create a target for a custom board, for example:
+- A simple way would be to identify an existing board which is close to your custom board and modify the entries in this boards hal file.
+- A more advanced way would be to create a new target as you find them for the existing boards. This requires more work and additional skills.
+
+The suggested procedure is simple and makes it easier to update the code base by seperating out changes:
+ 
+1. Identify a target which is close to your new board. Example: The `rx-diy-board01-f103cb` target may be it.
+    1. Note: the only condition when picking a target is that you will need to choose a target which uses the same MCU as your custom board, in this example it is assumed that you're using the STM32F103CBXX on your custom board.
```

Comment:
maybe use - instead of 1. (dito below)
maybe wirte Note: The ...

Reply:
Done